basics/answers.md

```diff
@@ -156,7 +156,16 @@ isn't exactly like mine but it works just fine it's ok, and you can
     print(message, "!!!")
     print(message, "!!!")
     ```
-
+3. Here, we create  a new variable `palindrome_check`.We use this variable to store the  reversed  string(of`palindrome_input`). Then we check if the string that was given(`palindrome_input`)
+   and the reversed sting(`palindrome_check`) is the same using a simple `if loop`
+   ```python
+   palindrome_input=input("Type the number to check:") 
+   palindrome_check=palindrome_input[::-1] #Reverses the string
```

Comment:
Please explain `[::-1]` in `handy-stuff-strings.md` or add something like "Hint: google how to reverse a string" to the exercise.